changelog_5c1ce29804286304a71d9ba4.md

```diff
@@ -2,6 +2,18 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of September 30, 2019
+  
+- [The latest release of the Semaphore Agent](https://github.com/semaphoreci/agent/pull/72)
```

Comment:
Let's state the exact version number (time-agnostic) instead of "The latest release of".